NativeApp/src/components/User/container.js

```diff
@@ -46,6 +47,25 @@ export default Container => class extends Component {
     this.sub.remove();
   }
 
+  sortingEvents = (events) => {
+    let eventArray = [];
+    let approvalArray = [];
+    let awaitApprovalArray = [];
+
+    approvalArray.push(events.filter(event => event.eventStatus.description === 'Approved'));
+    awaitApprovalArray.push(events.filter(event => event.eventStatus.description === 'Awaiting approval'));
```

Comment:
I'd call these something more like `approvedEvents` and `pendingEvents`. If "awaitApproval" is the terminology you've used elsewhere in the app then ignore me. :)